### ISSUE TITLE
Remove www directory

### ISSUE THREAD

**Issue by alex-konovalov** [gap-packages/hap#25]
This is a follow-up of #24. 
 
I have published 1.24 with ReleaseTools, but had to tweak `gh-pages` manually as explained at https://github.com/gap-packages/hap/wiki/HAP-wiki. 

A small TODO is to add missing .css and .js files to manuals (see error messages on the wiki page above) - see also #14 

A larger project is to completely remove `www`, as was outlined in #24.

**Comment by grahamknockillaree** [comment]
I've added manual.js to Hap1.25/doc/ and I've adjusted ./clean so that it no longer deletes the .js file.

Removing the www directory is a long term aim. But there are lot's of examples of HAP code that, at present, are documented only in the www directory.  So I'll close this request.

**Comment by alex-konovalov** [comment]
@grahamknockillaree OK, we can live with this - e.g. https://gap-packages.github.io/hap/ links to `www` for package manual and it works. But this makes using ReleaseTools not straightforward - I've wrote which tweaks I had to do last time at https://github.com/gap-packages/hap/wiki/HAP-wiki. I suggest that when you will be ready to publish next release, you will ask me again and I will publish it for you.
